v0/destinations/mailchimp/utils.js

```diff
@@ -0,0 +1,87 @@
+const axios = require("axios");
+const md5 = require("md5");
+const logger = require("../../../logger");
+const { CustomError } = require("../../util");
+
+const getMailChimpEndpoint = (datacenterId, audienceId) => {
+  const mailChimpApi = "api.mailchimp.com";
+  const listsUrl = `https://${datacenterId}.${mailChimpApi}/3.0/lists`;
+  return `${listsUrl}/${audienceId}`;
+};
+
+// Converts to upper case and removes spaces
+const filterTagValue = tag => {
+  const maxLength = 10;
+  let newTag = tag.replace(/[^\w\s]/gi, "");
+  if (newTag.length > maxLength) {
+    newTag = newTag.slice(0, 10);
+  }
+  return newTag.toUpperCase();
+};
+
+const checkIfMailExists = async (apiKey, datacenterId, audienceId, email) => {
+  // ref: https://mailchimp.com/developer/marketing/api/list-members/get-member-info/
+  let status = false;
+  if (!email) {
```

Comment:
This check is not need because of the check we are already doing at L196 in transform.js

Reply:
done